### ISSUE TITLE
SyntaxError: Unexpected strict mode reserved word

### ISSUE THREAD

**Issue by sustrik** [sweet-js/sweet.js#563]
Following the tutorial:

1. npm install sweet.js
2. test.js = syntax hi = function(ctx) ... and so on
3. node_modules/.bin/sjs test.js

/home/sustrik/node_modules/sweet.js/dist/sweet.js:59
  let options_764 = arguments.length <= 1 || arguments[1] === undefined ? {} :
  ^^^
SyntaxError: Unexpected strict mode reserved word
    at Module._compile (module.js:439:25)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/home/sustrik/node_modules/sweet.js/bin/sjs:21:15)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)

**Comment by akalenda** [comment]
Hi, I also had this problem. It turned out to be due to the way NodeJS was installed on my system.

If you look inside SweetJS's `package.json`, you'll see that the minimum version of NodeJS it requires is `5.0`. Type `nodejs -v` to see what version you have installed. In my case it was somewhere around `v.0.6` or so. You probably have to finagle the package manager. Check the [NodeJS instructions](https://nodejs.org/en/download/package-manager/) for this.

The following fixed it for me:

```
npm uninstall sweet.js
sudo apt-get remove nodejs
curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -
sudo apt-get install -y nodejs
npm install sweet.js
```

**Comment by gabejohnson** [comment]
@disnet It might make sense to add a `.npmrc` file with `engine-strict` set to true. This is the second time I've seen someone stumble on this one.

**Comment by gabejohnson** [comment]
Nevermind. I see that a user has to set `engine-strict` in their configuration. But a `preinstall` script could be used to match the installed version against `package.json`.

**Comment by disnet** [comment]
Fixed by #585
